tmt/steps/provision/minute.py

```diff
@@ -69,18 +70,55 @@ class ProvisionMinute(tmt.steps.provision.ProvisionPlugin):
             image: 1MT-Fedora-32
             flavor: m1.large
 
-    Available images and flavors can be listed using '1minutetip list'
-    and '1minutetip list-flavors'.
+    Available images and flavors can be listed using 'tmt run provision --how minute --list'
+    and 'tmt run provision --how minute --list-flavors'.
     """
 
     # Guest instance
     _guest = None
 
+    # API url for openstack gateway
+    api_url = ""
+
     # Supported methods
     _methods = [
         tmt.steps.Method(name='minute', doc=__doc__, order=50),
         ]
 
+    def _populate_api_url(self):
+        """ Get api url from 1minutetip script """
+        # Read API URL from 1minutetip script
+        try:
+            self.debug(f"Get the API URL from '{SCRIPT_PATH}'.")
+            script_content = self.read(SCRIPT_PATH)
+            match = re.search(API_URL_RE, script_content)
+            if not match:
+                raise tmt.utils.ProvisionError(
+                        f"Could not obtain API URL from '{SCRIPT_PATH}'.")
+            self.api_url = match.group('url')
+            self.debug('api_url', self.api_url, level=3)
+        except tmt.utils.FileError:
+            raise tmt.utils.ProvisionError(
+                f"File '{SCRIPT_PATH}' not found. Please install 1minutetip.")
+
+    def _filter_images_list_output(self, image_list_raw):
+        """ Prepare raw image list to terminal """
+        filter_out = ('new', 'obsolete', 'invalid')
+        image_list = list(filter(lambda line:
+            not line.endswith(filter_out) and line.startswith('1MT-'),
+            image_list_raw.splitlines()
+        ))
+        image_list.sort()
+
+        return image_list
+
+    def _print_images_list(self, image_list):
+        os_groups = ('Fedora', 'CentOS', 'RHEL-5' 'RHEL-6', 'RHEL-7', 'RHEL-ALT', 'RHEL-8', 'RHEL-9')
+        for os_group in os_groups:
+            os_list = list(filter(lambda item: os_group in item, image_list))
+            print("\n".join(os_list))
+            sys.stderr.write("-------------------------\n")
```

Comment:
How about using verbosity level for "group separation"?

Reply:
I see an empty group (empty line between dashes). Can it be filtered?

Reply:
This is just same output, as 1minutetip list provides... so it's 1:1 even with delimiters on stderr.
Where do you see empty group? Like some python formating just add whitespaces?

Reply:
I see this...
```
1MT-CentOS-7
1MT-CentOS-8
-------------------------

-------------------------
1MT-RHEL-7-pulp-prod
1MT-RHEL-7-pulp-stage
```

Reply:
Ok. Bug in code. Missing , character.